addons/event_booth_sale/models/sale_order_line.py

```diff
@@ -92,11 +92,16 @@ def _get_sale_order_line_multiline_description_sale(self):
 
     def _get_display_price(self):
         if self.event_booth_pending_ids and self.event_id:
-            company = self.event_id.company_id or self.env.company
-            currency = company.currency_id
-            total_price = sum([booth.price for booth in self.event_booth_pending_ids])
-            return currency._convert(
-                total_price, self.order_id.currency_id,
-                self.order_id.company_id or self.env.company.id,
-                self.order_id.date_order or fields.Date.today())
+            pricelist_id = self.order_id.pricelist_id.id
+            return sum(super(SaleOrderLine,
+                             self.with_context(
+                                 pricelist=pricelist_id,
+                                 record_being_sold=booth._origin.booth_category_id.id,
+                                 quantity=1)  # Assume there are no duplicate booth -> quantity = 1
+                             )._get_display_price() for booth in self.event_booth_pending_ids)
```

Comment:
I would do something to make this a bit more readable, perhaps like this?
+2 lines
```suggestion
            return sum(
                super(SaleOrderLine,
                      self.with_context(pricelist=pricelist_id,
                                        record_being_sold=booth._origin.booth_category_id.id,
                                        quantity=1)  # Assume there are no duplicate booth -> quantity = 1
                )._get_display_price() 
                for booth in self.event_booth_pending_ids
            )
```
or, +1line 
```suggestion
            return sum(super(SaleOrderLine,
                             self.with_context(
                                 pricelist=pricelist_id,
                                 record_being_sold=booth._origin.booth_category_id.id,
                                 quantity=1)  # Assume there are no duplicate booth -> quantity = 1
                             )._get_display_price()
                       for booth in self.event_booth_pending_ids)
```

Reply:
Thanks for the proposal. I have taken a mix of the 2. I think indeed that having the "for" aligned with the elements we sum is the most important thing, it is easier to read.